tests/lora/utils.py

```diff
@@ -278,32 +291,23 @@ def test_simple_inference_with_text_lora_and_scale(self):
                         check_if_lora_correctly_set(pipe.text_encoder_2), "Lora not correctly set in text encoder 2"
                     )
 
-            output_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
+            output_lora = pipe(**inputs, generator=torch.manual_seed(0))[0]
             self.assertTrue(
-                not np.allclose(output_lora, output_no_lora, atol=1e-3, rtol=1e-3), "Lora should change the output"
+                not np.allclose(output_lora, output_no_lora, atol=1e-3, rtol=1e-3),
+                "Lora should change the output",
```

Comment:
is this done by our linter? I saw a trailing comma added in multiple places, ok here! but I think we do not need to do that in the future for function calls,

Reply:
So earlier, I had introduced a change for taking expected_atol and expected_rtol as parameters in some methods that were later not required after making text encoder related changes. This increased the line length and caused the linter to split into multiple lines. I'll try to revert these when I'm next awake

Reply:
Fixed